IBM-Cloud/ibm-developer-extension-vscode
# test: Added vscode integration tests

# Context
This PR is to add integration test suite that includes testing the existing commands released for the extension as well as the new commands added in #42 

To prevent merge conflicts this PR will use branch #42 as the base. 

# Callouts
Additional housekeeping are added. 

1. Removed `npm test` script from the git commit hooks
2. Included detect-secrets npm script and travis build.
3. Exclude `npm test` script from `script` stage in the travis build.

# Steps to Test
1. Set the API token to the environment variable `IBMCLOUD_API_KEY`
2. Run the command `npm test` 
3. Verify that the tests pass successfully.
